src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "You are playing the Lottery game!!!" + "\n{"
+                + " ball color: '" + color + '\''
+                + ", ball number: " + number + '}';
```

Comment:
The toString() method should be used to represent the state of the object, not to display a game message. It would be better to move the game message 'You are playing the Lottery game!!!' to another place, for example, to the method where you are using this object.